summary/summary.go

```diff
@@ -70,7 +71,7 @@ func Summary(c *k8s.Client, o Options) error {
 	// create a client
 	conn, err := grpc.Dial(gRPC, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	if err != nil {
-		return errors.New("could not connect to the server. Possible troubleshooting:\n- Check if discovery engine is running\n- Create a portforward to discovery engine service using\n\t\033[1mkubectl port-forward -n explorer service/knoxautopolicy --address 0.0.0.0 --address :: 9089:9089\033[0m\n[0m")
+		return nil, errors.New("could not connect to the server. Possible troubleshooting:\n- Check if discovery engine is running\n- Create a portforward to discovery engine service using\n\t\033[1mkubectl port-forward -n <namespace> service/knoxautopolicy --address 0.0.0.0 --address :: 9089:9089\033[0m\n[0m")
```

Comment:
Since port-forwarding is happening internally. We don't have to mention port-forward explicitly.
so the error message could be like this 
```console
"could not connect to the server. Possible troubleshooting:\n- Check if discovery engine is running\n- kubectl get po -n accuknox-agents"
``` 
this is enough I guess

Reply:
ok

Reply:
@yasin-cs-ko-ak updated the err msg